contracts/Sweep/TransferApprover/VestingApprover.sol

```diff
@@ -247,14 +199,6 @@ contract VestingApprover is ITransferApprover, Ownable {
         return vestingSchedules[beneficiary];
```

Comment:
line 197 is not necessary

Reply:
is this function necessary? we have a public variable named vestingSchedules that can be consulted

Reply:
I removed that funcition